python/main.py

```diff
@@ -1,139 +1,176 @@
 import json
+import settings
+import ynab_client
 
-from flask import Flask, render_template, request, jsonify, redirect
+from flask import Flask, request, jsonify, redirect
 from datetime import datetime
 from dateutil.parser import parse
 from decimal import Decimal
 
 from pynYNAB.schema.budget import Transaction
 
-import settings
-import ynab_client
-
 app = Flask('Monzo to YNAB')
 app.config['DEBUG'] = settings.flask_debug
 
-if settings.sentry_dsn:
-    from raven.contrib.flask import Sentry
-    sentry = Sentry(app)
 
 @app.route('/')
 def route_index():
     return redirect("https://github.com/scottrobertson/monzo-to-ynab", code=302)
 
+
 @app.route('/webhook', methods=['POST'])
 def route_webhook():
-    expected_delta = 0
+    ynab_client.init()
```

Comment:
Good point, I'll make sure it's a singleton and only inits if it's not already initialized.